src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionService.kt

```diff
@@ -37,17 +43,45 @@ class LicenceConditionService(
     val entityStandardPssConditions = request.standardPssConditions.transformToEntityStandard(licenceEntity, "PSS")
 
     val username = SecurityContextHolder.getContext().authentication.name
+    val createdBy = communityOffenderManagerRepository.findByUsernameIgnoreCase(username)
+      ?: throw RuntimeException("Staff with username $username not found")
 
     val updatedLicence = licenceEntity.copy(
       standardConditions = entityStandardLicenceConditions + entityStandardPssConditions,
       dateLastUpdated = LocalDateTime.now(),
       updatedByUsername = username,
     )
+
+    /*
+    When the function is called, the standard conditions are updated in line with the current policy version but
+    the licence will retain the version of the policy it was created with because any additional conditions will be on a previous
+    version and not updated. These additional conditions can only be updated via the vary journey.
+    */
+    val currentPolicyVersion = licencePolicyService.currentPolicy().version
+
+    val changes = mapOf(
+      "typeOfChange" to "update",
+      "condition" to "standard",
+      "changes" to emptyMap<String, Any>()
+    )
+
     licenceRepository.saveAndFlush(updatedLicence)
+
+    auditEventRepository.saveAndFlush(
+      AuditEvent(
+        licenceId = licenceId,
+        username = username,
+        fullName = "${createdBy.firstName} ${createdBy.lastName}",
+        summary = "Standard conditions updated to policy version $currentPolicyVersion for ${licenceEntity.forename} ${licenceEntity.surname}",
+        detail = "ID ${licenceEntity.id} type ${licenceEntity.typeCode} status ${licenceEntity.statusCode.name} version ${licenceEntity.version}",
+        changes = changes
+      )
+    )
   }
 
   /**
    * Add additional condition. Allows for adding more than one condition of the same type
+   * TODO - This function is only called for subsequent calls to upload a map as part of an additional condition only - needs to be refactored
```

Comment:
What do you mean by needs to be refactored? do we have a ticket for that?

Reply:
I will run it by the team and then raise a ticket for it if necessary following that - removed